### PR TITLE
[2.13.x] DDF-04905 Prevent ffmpeg access control exception

### PR DESCRIPTION
**Pull request by ahoffer**
#### What does this PR do?
Wraps the call to start an OS process (execute ffmpeg) in a do-privilege

#### Who is reviewing it? 
@austinsteffes @tyler30clemens @peterhuffer @brjeter 

#### Ask 2 committers to review/merge the PR and tag them here.
@blen-desta
@emmberk

#### How should this be tested?
@peterhuffer Can you hero this? You've had the best luck reproducing the problem during replication.

#### Any background context you want to provide?
None

#### What are the relevant tickets?
Fixes: #4905 

#### Screenshots
N/A
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java [hunk 1]
```diff
@@ -352,8 +355,7 @@ private String durationToString(final Duration duration) {
   }
 
   private DefaultExecuteResultHandler executeFFmpeg(
-      final CommandLine command, final int timeoutSeconds, final PumpStreamHandler streamHandler)
-      throws IOException {
```
Comment (peterhuffer) [inline catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java]: ❓ It looks like this IOException was being bubbled up. Do we need to do an instance of check on the privileged action caused by exception and rethrow if it is an IOException?

**Comment by peterhuffer** [comment]
✔️ I was able to verify that this change removed the access control exception when ingesting wmv and mpg resources using replication.

**Comment by peterhuffer** [comment]
✔️ Hero with latest changes successful. No access control exceptions when running replication with wmv and mpg resources.

**Comment by ahoffer** [comment]
build now

**Comment by peterhuffer** [comment]
We should cleanup the security permissions granted to bundles related to the ffmpeg library. https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/security/default.policy#L402
I don't think anyone else besides the video thumbnail plugin should need it now (unless someone else is using the library directly).

**Comment by ahoffer** [comment]
@peterhuffer Permissions cleanup addressed in https://github.com/codice/ddf/issues/4910

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Pull request merged**